### DISCUSSION TITLE
ELN Consortium meeting 6

### DISCUSSION THREAD

**Discussion by NicolasCARPi** [TheELNConsortium/TheELNFileFormat#73] [General]
September 10th 4PM.

**Comment by NicolasCARPi** [comment]
* figure out string or int for `contentSize` https://github.com/TheELNConsortium/TheELNFileFormat/pull/79#discussion_r1686582178

**Comment by manulera** [comment]
Hello all,

@NicolasCARPi invited me to give a short 10 min presentation about [ShareYourCloning](https://github.com/manulera/ShareYourCloning), a project I am working on to allow researchers to plan DNA engineering, and keep FAIR records of the provenance of engineered DNA sequences.

The project has two main focus areas:

* Developing a [web application](https://shareyourcloning.org/) for researchers to plan how they are going to make new plasmids or engineer genomes.
* Developing a [data model](https://github.com/genestorian/ShareYourCloning_LinkML/) to capture that provenance, so that it can be stored, queried and shared with others.

The best way to get a feeling of what the project is about and where it is right now is to watch this [15 min video](https://www.youtube.com/watch?v=n0hedzvpW88) where I briefly explain the principle and do a small demo of the app. I will anyways present it in the meeting.

We would like to develop an integration with eLabFTW where the users can read sequences from their database, clone using ShareYourCloning's web application, and save the sequence produced back in eLabFTW. I developed a demo integration that is mentioned in [this issue](https://github.com/manulera/ShareYourCloning/issues/8). The functionality is shown in two videos included in the issue:

https://github.com/manulera/ShareYourCloning/assets/22526102/843d1114-b8b2-4420-aae3-a7992a7e349f

https://github.com/manulera/ShareYourCloning/assets/22526102/ac9f3c7b-5f9c-471a-bb3f-c7131c056b0b

**Comment by NicolasCARPi** [comment]
Hey all, here is the zoom link for tomorrow:

https://pwpush.com/p/boagyvcq3nvb2fun_s0/r

See you!

**Reply by NicolasCARPi** [reply]
hXXps://zoom.us/j/94521789192?pwd=oARTFetTbDtrXoDIw1cR5v9oK9JZY4.1

**Comment by NicolasCARPi** [comment]
# Minutes

## Present

* Florian : Jülich : SampleDB ELN
* Nico Brandt : Karlsruhe : Kadi4Mat ELN-like System
* Manideep : Karlsruhe : Kadi4Mat ELN-like System
* Lauritz Bussfeld : maintainer of research data repo at Edinburgh : using ckan.org
* Joe Davies : Nottingham, UK, AI4Green ELN
* Manuel Lera Ramirez : presentation of https://shareyourcloning.netlify.app/
* Nicolas CARPI & Sydney Koke : eLabFTW - Deltablot

## Discussion

contentSize: keep as as a string, but no units, bytes size, need to put in spec


some things can be only one, such as "genre". But in some contexts it can be several, so we should not assume there is only one thing, but if the importing party doesn't support multiple values for this attribute, either throw an error, or emit a warning and import the first one found.

signatures (https://github.com/TheELNConsortium/TheELNFileFormat/issues/56): use .well-known/keys.json, which is json-ld and can describe several keys (such as past keys), and different keys formats.


Lauritz: they use datacite to describe datasets in CKAN (https://github.com/TheELNConsortium/TheELNFileFormat/discussions/82)

Nico B. : for main text: description: use a TextObject with encodingFormat so we can discriminate markdown/html/whatever

So the value for `description` of a `Dataset` could be a string, or a TextObject with more precise description.



next meeting 12th of Nov: https://github.com/TheELNConsortium/TheELNFileFormat/discussions/83

**Reply by manulera** [reply]
This is my presentation about [ShareYourCloning](https://shareyourcloning.org/)

 [ELN_consortium_10092024.pdf](https://github.com/user-attachments/files/16958892/ELN_consortium_10092024.pdf)
